src/core/ext/resolver/dns/native/dns_resolver.c

```diff
@@ -71,9 +72,9 @@ typedef struct {
   /** pending next completion, or NULL */
   grpc_closure *next_completion;
   /** target result address for next completion */
```

Comment:
I think this comment is still accurate.  This is in fact still the resolver's result; it's just that the result is now returned as a `grpc_channel_args` object instead of a `grpc_resolver_result` object.